packages/salesforce-adapter/src/filters/organization_wide_sharing_defaults.ts

```diff
@@ -32,7 +32,35 @@ const log = logger(module)
 
 const ORGANIZATION_SETTINGS_INSTANCE_NAME = 'OrganizationSettings'
 
-const enrichTypeWithFields = async (client: SalesforceClient, type: ObjectType): Promise<void> => {
+/*
+* These fields are not multienv friendly
+* */
+const FIELDS_TO_IGNORE = [
+  'DailyWebToCaseCount',
+  'DailyWebToLeadCount',
+  'InstanceName',
+  'IsSandbox',
+  'LastWebToCaseDate',
+  'LastWebToLeadDate',
+  'MonthlyPageViewsEntitlement',
+  'MonthlyPageViewsUsed',
+  'OrganizationType',
+  'SelfServiceCaseSubmitRecordTypeId',
+  'SelfServiceEmailUserOnCaseCreation‍TemplateId',
+  'SelfServiceNewCommentTemplateId',
+  'SelfServiceNewPassTemplateId',
+  'SelfServiceNewUserTemplateId',
+  'SelfServiceSolutionCategoryStartNodeId',
+  'TrialExpirationDate',
+  'WebToCaseAssignedEmailTemplateId',
+  'WebToCaseCreatedEmailTemplateId',
+  'WebToCaseDefaultCreatorId',
+]
+
+const enrichTypeWithFields = async (
+  client: SalesforceClient,
+  type: ObjectType,
+  fieldsToIgnore: Set<string>): Promise<void> => {
```

Comment:
Super Nit (style): This will create a cleaner commit next time someone adds a new parameter. One line instead of 2.
```suggestion
  fieldsToIgnore: Set<string>,
  ): Promise<void> => {
```

Reply:
Ack